### ISSUE TITLE
LET cubes cannot be read if suffix dot is missing

### ISSUE THREAD

**Issue by nbassler** [pytrip/pytrip#296]
```
cube_001_subdosemlet.dos
cube_001_subdosemlet.hed
```

trying to read them fails:

```
bassler@baslup:~/Projects/dasu01$ python3
Python 3.5.3 (default, Jan 19 2017, 14:11:04) 
[GCC 6.3.0 20170118] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytrip as pt
>>> pt.__version__
'1.3.8'
>>> l = pt.LETCube()
>>> l
<pytrip.let.LETCube object at 0x7fa68b4bae48>
>>> l.read("cube_001_sub")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 677, in read
    self.read_trip_data_file(path)
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 750, in read_trip_data_file
    raise IOError("Could not find file " + path)
OSError: Could not find file cube_001_sub
>>> l.read("cube_001_subdosemlet")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 677, in read
    self.read_trip_data_file(path)
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 750, in read_trip_data_file
    raise IOError("Could not find file " + path)
OSError: Could not find file cube_001_subdosemlet
>>> l.read("cube_001_subdosemlet.dos")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 677, in read
    self.read_trip_data_file(path)
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 743, in read_trip_data_file
    header_file_path = self.discover_file(header_file_name)
  File "/home/bassler/.local/lib/python3.5/site-packages/pytrip/cube.py", line 484, in discover_file
    gzip_file_name = file_name + ".gz"
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
>>> 
```

renaming into 
```
cube_001_sub.dosemlet.dos
cube_001_sub.dosemlet.hed
```
works however.

**Comment by grzanka** [comment]
Is it a bug ?

I thought that the extension should have dot in it, see:
https://github.com/pytrip/pytrip/blob/master/pytrip/let.py#L36

```
    data_file_extension = ".dosemlet.dos"
    header_file_extension = ".dosemlet.hed"
```

**Comment by nbassler** [comment]
yes, but I realize now, that trip also readly produces them without leading dot. Both may occur. :-(

**Comment by grzanka** [comment]
It can be fixed, but I'd suggest a consistent approach - what about .dos, .ctx and .vdx ?
May they also occur without dots ?

**Comment by nbassler** [comment]
nope.

Generally, pytrip throws very strange error messages when trying to read a filename which is not formatted as expected. A "file not found <foobar>" would be more helpful than random error messages about ".gz" cannot be appended to NoneType or so.

**Comment by nbassler** [comment]
This is so super annoying:

```
bassler@grgalt:~/test$ ls
TST000000.ctx  TST000000.hed  TST000000.phys.dos  TST000000.vdx

bassler@grgalt:~/test$ python
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.

>>> import pytrip as pt
>>> d = pt.DosCube()
>>> d.read("TST000000.phys.dos")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bassler/.local/lib/python2.7/site-packages/pytrip/cube.py", line 733, in read
    self.read_trip_data_file(path)
  File "/home/bassler/.local/lib/python2.7/site-packages/pytrip/cube.py", line 806, in read_trip_data_file
    raise IOError("Could not find file " + path)
```

**Comment by nbassler** [comment]
I guess it is because it expects a phys.hed as well. Could throw a clearer message then.

**Comment by nbassler** [comment]
@grzanka can we get this one fixed for 2.0.0?
